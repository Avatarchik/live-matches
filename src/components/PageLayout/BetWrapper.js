import React from 'react';
import Content from './Content';

const BetWrapper = () => {
  return (
    <div id="content">
      <article>
        <h1>Live matches</h1>
        <p className="preamble">
          Here is a list of matches that are live right now.
        </p>
        <aside>
          <h2>Live betting</h2>
          <p>
            Place your bets as the action unfolds. We offer a wide selection of
            live betting events and you can place both single and combination
            bets.
          </p>
          <p>
            You will be able to see an in-play scoreboard with the current
            result and match stats, while on selected events you will also be
            able to watch the action live with Unibet TV on the desktop site.
          </p>
        </aside>
        <Content />
      </article>
    </div>
  );
};
export default BetWrapper;
