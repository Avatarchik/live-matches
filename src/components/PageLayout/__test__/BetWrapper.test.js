import React from 'react';

import { shallow } from 'enzyme';
import BetWrapper from '../BetWrapper';

describe('Test suite for Navbar Component', () => {
  // create a UI snapshot to indentify UI changes or design regressions

  it('render correctly text component', () => {
    const component = shallow(<BetWrapper />);
    expect(component).toMatchSnapshot();
  });
});
