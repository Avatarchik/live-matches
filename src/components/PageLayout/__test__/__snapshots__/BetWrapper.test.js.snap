// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for Navbar Component render correctly text component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BetWrapper />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <article>
        <h1>
          Live matches
        </h1>
        <p
          className="preamble"
        >
          Here is a list of matches that are live right now.
        </p>
        <aside>
          <h2>
            Live betting
          </h2>
          <p>
            Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.
          </p>
          <p>
            You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.
          </p>
        </aside>
        <Content />
      </article>,
      "id": "content",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Live matches
          </h1>,
          <p
            className="preamble"
          >
            Here is a list of matches that are live right now.
          </p>,
          <aside>
            <h2>
              Live betting
            </h2>
            <p>
              Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.
            </p>
            <p>
              You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.
            </p>
          </aside>,
          <Content />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Live matches",
          },
          "ref": null,
          "rendered": "Live matches",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Here is a list of matches that are live right now.",
            "className": "preamble",
          },
          "ref": null,
          "rendered": "Here is a list of matches that are live right now.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                Live betting
              </h2>,
              <p>
                Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.
              </p>,
              <p>
                You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.
              </p>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Live betting",
              },
              "ref": null,
              "rendered": "Live betting",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.",
              },
              "ref": null,
              "rendered": "Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.",
              },
              "ref": null,
              "rendered": "You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.",
              "type": "p",
            },
          ],
          "type": "aside",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "article",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <article>
          <h1>
            Live matches
          </h1>
          <p
            className="preamble"
          >
            Here is a list of matches that are live right now.
          </p>
          <aside>
            <h2>
              Live betting
            </h2>
            <p>
              Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.
            </p>
            <p>
              You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.
            </p>
          </aside>
          <Content />
        </article>,
        "id": "content",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Live matches
            </h1>,
            <p
              className="preamble"
            >
              Here is a list of matches that are live right now.
            </p>,
            <aside>
              <h2>
                Live betting
              </h2>
              <p>
                Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.
              </p>
              <p>
                You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.
              </p>
            </aside>,
            <Content />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Live matches",
            },
            "ref": null,
            "rendered": "Live matches",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Here is a list of matches that are live right now.",
              "className": "preamble",
            },
            "ref": null,
            "rendered": "Here is a list of matches that are live right now.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  Live betting
                </h2>,
                <p>
                  Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.
                </p>,
                <p>
                  You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Live betting",
                },
                "ref": null,
                "rendered": "Live betting",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.",
                },
                "ref": null,
                "rendered": "Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.",
                },
                "ref": null,
                "rendered": "You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.",
                "type": "p",
              },
            ],
            "type": "aside",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "article",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
